api/internal/target/kusttarget.go

```diff
@@ -45,6 +45,7 @@ func NewKustTarget(
 	rFactory *resmap.Factory,
 	pLdr *loader.Loader) *KustTarget {
 	pLdrCopy := *pLdr
+	pLdrCopy.SetConfig()
 	pLdrCopy.SetWorkDir(ldr.Root())
```

Comment:
I think it might be better to create a `Loader.DeepCopy` method that combines lines 47-49 (and fold your `Loader.SetConfig` into it). WDYT?

Reply:
Good idea, I've implemented it :+1: